hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/stream/StreamingException.java

```diff
@@ -24,7 +24,7 @@
  */
 public class StreamingException extends RuntimeException {
 
-  public StreamingException(InterruptedException ex) {
+  public StreamingException(Exception ex) {
```

Comment:
With the new `StreamingException(Throwable)` constructor, I think this one is no longer needed.